src/Color/ColorConverter.php

```diff
@@ -173,7 +173,7 @@ private function convertPlain($color, array &$PDFAXwarnings = [])
 		$c = false;
 
 		if (preg_match('/^[\d]+$/', $color)) {
-			$c = [static::MODE_GRAYSCALE, $color]; // i.e. integer only
+			$c = [static::MODE_GRAYSCALE, (float) $color]; // i.e. integer only
```

Comment:
This could be replaced by a cast to int, but a cast to int or float needs to happen.